docs/templates/getting-started/faq.md

```diff
@@ -146,7 +146,25 @@ layer_output = get_3rd_layer_output([X, 0])[0]
 layer_output = get_3rd_layer_output([X, 1])[0]
 ```
 
-Another more flexible way of getting output from intermediate layers is to use the [functional API](/getting-started/functional-api-guide).
+Another more flexible way of getting output from intermediate layers is to use the [functional API](/getting-started/functional-api-guide). For example, after fitting an autoencoder on MNIST:
+
+```python
+inputs = Input(shape=(784,))
+first = Dense(512)(inputs)
+second = Dense(32)(first)
+third = Dense(512)(second)
+outputs = Dense(784)(third)
+model = Model(input=inputs, output=outputs)
+model.compile(optimizer='rmsprop', loss='mse')
+model.fit(X, X, ...)
+```
```

Comment:
Why not something even simpler:

```python
inputs = Input(shape=(784,))
encoded = Dense(32, activation='relu')(inputs)
decoded = Dense(784)(encoded)
model = Model(input=inputs, output=decoded)

encoder = Model(input=inputs, output=encoded)
```
Btw lack of activation function means that the stack of layers acts as a single linear transformation, which is probably not what you want... Also for an autoencoder on MNIST it would be better to use a `sigmoid` activation on the decoder layer and a `binary_crossentropy` loss, but that's a nitpick.

Even though it's a toy example... one thing I've learned is that people will actually copy and try to use toy examples from the docs that were meant purely for illustrating something, so they should be written carefully.